api/chalicelib/core/tags.py

```diff
@@ -0,0 +1,49 @@
+import schemas
+from chalicelib.utils import helper
+from chalicelib.utils import pg_client
+
+
+def create_tag(project_id: int, data: schemas.TagCreate) -> int:
+    query = """
+    INSERT INTO public.tags (project_id, selector, ignore_click_rage, ignore_dead_click)
+    VALUES (%(project_id)s, %(selector)s, %(ignore_click_rage)s, %(ignore_dead_click)s)
+    RETURNING tag_id;
+    """
+
+    # Remove project_id if any, to avoid a clash in the query
+    data.pop('project_id', None)
+
+    with pg_client.PostgresClient() as cur:
+        query = cur.mogrify(query, {'project_id': project_id, **data})
+        cur.execute(query)
+        row = cur.fetchone()
+
+    return row['tag_id']
+
+
+def list_tags(project_id: int):
+    query = """
+    SELECT tag_id, selector, ignore_click_rage, ignore_dead_click
```

Comment:
name is missing

Reply:
Fixed in https://github.com/openreplay/openreplay/pull/1834/commits/4fd84ae32fce0b69019b31df9b81d999b22a4c4f